bats/tests/helpers/utils.bash

```diff
@@ -1,6 +1,7 @@
 is_true() {
     # case-insensitive check; false values: '', '0', 'no', and 'false'
-    if [[ "${1,,}" =~ ^(0|no|false)?$ ]]; then
+    local value="$(echo "$1" | tr '[A-Z]' '[a-z]')"
```

Comment:
Thank you. The `${VAR,,}` form isn't supported on macos bash, and it's poorly documented at any rate.

Reply:
It is a bash 4 thing